assets/css/src/components-form-editor/_template-selection.scss

```diff
@@ -1,3 +1,22 @@
-.template-selection {
-  margin: $grid-gap-large;
+.mailpoet-templates {
+  display: grid;
+  grid-gap: 16px;
```

Comment:
Why do you use hardcoded values rather than already predefined values in this file? Also, some values are dependant on others so we should calculate them rather than hardcoding them.